pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -172,9 +174,8 @@ pub mod pallet {
 		#[pallet::weight(<T as Config>::WeightInfo::associate_account())]
 		pub fn associate_account(
 			origin: OriginFor<T>,
-			account: AccountIdOf<T>,
+			req: AssociateAccountRequest,
```

Comment:
[PLACEHOLDER] The extrinsic weight must be updated with the new value once the benchmarks are through